lib/game.rb

```diff
@@ -0,0 +1,154 @@
+
+class Game
+    attr_reader :player, :frame, :roll, :score, :count_next, :pins_rest , :list_boundaries, :message
+
+    def initialize(player, pins_rest = 10, frame = 1, roll = 0, score = 0, count_next = 0)
+      @player, @frame, @roll, @this_roll_count, @this_roll_bonus = capitalize_name(player), frame, roll, 0, 0
```

Comment:
assigning so many variables on a single line like this is not easy to read, particularly when some of the variables have similar names/values, would be more readable over multiple lines.

Reply:
Absolutely